client/dashboard/task-list/tasks/payments/square.js

```diff
@@ -43,7 +44,12 @@ class Square extends Component {
 	}
 
 	async connect() {
-		const { createNotice, options, updateOptions } = this.props;
+		const {
+			createNotice,
+			hasCbdIndustry,
+			options,
+			updateOptions,
+		} = this.props;
 		this.setState( { isPending: true } );
 
 		updateOptions( {
```

Comment:
I know this is unrelated to your changes, but since this is updating the wrong setting - can you fix?

Reply:
Hi @jeffstieler,
I fixed the issue in the commit [a8388ec](https://github.com/woocommerce/woocommerce-admin/pull/4061/commits/a8388ecc5c1ab8893508c32da580df637af769cc).

Also, there was a `validateDOMNesting` warning in the payment list (URL: `/wp-admin/admin.php?page=wc-admin&task=payments`) because we weren't adding only text inside a `p` tag. I added this change in the commit [dd22d5](https://github.com/woocommerce/woocommerce-admin/pull/4061/commits/7dd22d5676f2bec2699ba316d1db58dac3f7bcee).

Could you take a look at this change too?